pkg/process/build/build_test.go

```diff
@@ -0,0 +1,74 @@
+package process
+
+import (
+	"testing"
+)
+
+func TestBuild(t *testing.T) {
+
+	t.Run("Override should NOT be allowed for semanticVersion", func(t *testing.T) {
```

Comment:
Igjen litt forvirret, skal det være overwrite?

Reply:
Er det ikke lettere med en test-metode per test? Da kan man jo kjøre en og en i intellij, for eksempel..

Reply:
1 - [ ] overrite -> overwrite 
2- De kan kjøres en og en slik det er nå.